djl-serving/python-client/README.md

```diff
@@ -20,16 +19,13 @@ To register the model and make predictions:
 ```python
 import requests
 
-url = 'http://localhost:8080/models'
+# Register model
 params = {'url': 'https://resources.djl.ai/demo/pytorch/traced_resnet18.zip', 'engine': 'PyTorch'}
-res = requests.post(url, params=params)
+requests.post('http://localhost:8080/models', params=params)
 
 # Run inference
 url = 'http://localhost:8080/predictions/traced_resnet18'
-headers = {'Content-Type': 'application/octet-stream'}
-with open('/tmp/kitten.jpg', 'rb') as f:
```

Comment:
I'm trying to demo 2 formats. This is the example of binary input.

`files={'data': open('kitten.jpg', 'rb')}` below demo the multipart format.